### DISCUSSION TITLE
Cmake compilation failed: Wrong kit

### DISCUSSION THREAD

**Discussion by Edgar-ESTIA** [Estia-1a/projetGenieInfo_public#82] [Project Installation and Configuration] [answered]
**Bug description**
je rencontre un problème lors de la demande de compilation du Cmake, vous trouverez en pièce jointe une photo des message d'erreur. 
compilation du projet génie informatique
problème au niveau des librairies,  estia-image, getopt.

**Info about your config**

(Clic Preview to see the image below)
![Capture](https://user-images.githubusercontent.com/104554496/173312903-644097be-5b69-426b-bc72-422e62cd7c0a.PNG)
**Screen shot**
Add screenshots about the bug here.

**Comment by dhmmasson** [comment, marked as answer]
- Delete the build repository 
- Select the gcc kit again in vscode 
  - ctrl + shift + p
  - type : scan for kit
  - wait for the scan to complete
  - ctrl + shift + p 
  - type : select kit 
  - Select a GCC kit (gcc 4.6, gcc 9.x, gcc 11 or whichever version you have) 
- configure cmake
  - ctrl + shift + p
  - type : cmake configure
